packages/react-examples/README.md

```diff
@@ -1,47 +1,77 @@
 # @fluentui/react-examples
 
-**Examples and documentation for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+This package contains examples and documentation site content for `@fluentui/react` v8 and related packages, including `@fluentui/react-charting` and `@fluentui/react-experiments`.
```

Comment:
Here's my stab at expanding the docs some more...feel free to make further suggestions, or further changes after this merges.